docs/examples/en/controls/OrbitControls.html

```diff
@@ -192,7 +191,7 @@ <h3>[property:Float minDistance]</h3>
 
 		<h3>[property:Float minPolarAngle]</h3>
```

Comment:
I think you changed the wrong description here.

Reply:
ugh... thanks for the heads-up.